public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Cocktail Browser</title>
+    <title>IBA Cocktail Browser</title>
```

Comment:
Adding just because that's what it's finally turned into: IBA only - no longer any desire to expand beyond those.